site-service/src/test/java/uk/ac/ox/ndph/mts/site_service/SiteServiceImplIntegrationTests.java

```diff
@@ -85,4 +88,31 @@ void TestPostSite_WhenValidParentInput_Returns201AndId() throws Exception {
                 .perform(post("/sites").contentType(MediaType.APPLICATION_JSON).content(jsonString))
                 .andDo(print()).andExpect(status().isCreated()).andExpect(content().string(containsString("123")));
     }
+
+    @Test
+    void TestGetSites_ReturnsList() throws Exception {
+        // Arrange
+        final var org = new Organization()
+                .setName("CCO")
+                .addAlias("Root");
+        org.setId("this-is-my-id");
+        when(repository.getOrganizations()).thenReturn(List.of(org));
+        // Act + Assert
+        this.mockMvc
+                .perform(get("/sites").contentType(MediaType.APPLICATION_JSON))
```

Comment:
Can you move the route string to be a const on the class level and reuse through?

Reply:
Will do